daprdocs/content/en/developing-applications/building-blocks/workflow/howto-author-workflow.md

```diff
@@ -223,8 +269,97 @@ app.Run();
 
 {{% /codetab %}}
 
-{{< /tabs >}}
+{{% codetab %}}
 
+<!--python-->
+
+[In the following example](https://github.com/dapr/python-sdk/blob/master/examples/demo_workflow/app.py), for a basic Python order processing application using the Python SDK, your project code would include:
```

Comment:
This is not the case "for a basic Python order processing application using the Python SDK," is it  some other type of example, more like Hello World.